src/components/ListForm.js

```diff
@@ -0,0 +1,87 @@
+import styled from 'styled-components';
+
+const ListForm = ({ items, title, deleteTodo, toggleTodo }) => {
+  return (
+    <ListSection>
+      <ListTitle>{title}</ListTitle>
+      <ListWrapper>
+        {items.map((item, i) => (
+          <ListItemWrapper key={i} isDone={item.isDone}>
+            <ListItemText onClick={() => toggleTodo(item.text)}>
+              {item.text}
+            </ListItemText>
+            <DeleteBtn onClick={() => deleteTodo(item.text)}>❌</DeleteBtn>
+          </ListItemWrapper>
+        ))}
+      </ListWrapper>
+    </ListSection>
+  );
+};
+
+const ListSection = styled.section`
+  height: 40%;
+
+  border-bottom: 1px solid lightgrey;
+
+  & + & {
+    border: none;
+  }
+`;
+
+const ListTitle = styled.h3`
+  display: flex;
+  align-items: center;
+
+  margin: 0;
+  padding: 0.75rem;
+`;
+
+const ListWrapper = styled.div`
+  height: 9rem;
+
+  display: flex;
+  flex-direction: column;
+  overflow: auto;
+
+  &::-webkit-scrollbar {
+    width: 0.75rem;
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background: #d8d8d8;
+    background-clip: padding-box;
+
+    border-radius: 10px;
+    border: 0.25rem solid transparent;
+  }
+`;
+
+const ListItemWrapper = styled.div`
+  height: 2rem;
+
+  margin: 1rem 0 1rem 0;
```

Comment:
```suggestion
  margin: 1rem 0;
```

`margin`, `padding` 같은 속성은

- 값을 1개만 입력한 경우: 모든 방향
- 값을 2개만 입력한 경우: (상하), (좌우)
- 값을 4개 입력한 경우: (상), (우), (하), (좌) 

와 같은 방식으로 동작합니다

Reply:
상우하좌... 기억하겠습니다...

Reply:
시계 방향이라고 기억하시면 더욱 편합니다...!

Reply:
아니 그런 깊은 뜻이